biogeochem/EDCanopyStructureMod.F90

```diff
@@ -454,6 +473,12 @@ subroutine DemoteFromLayer(currentSite,currentPatch,i_lyr)
             
             cc_loss = currentCohort%excl_weight
             
+            leaf_c          = currentCohort%prt%GetState(leaf_organ,all_carbon_species)
+            store_c         = currentCohort%prt%GetState(store_organ,all_carbon_species)
```

Comment:
naive question - what are we including in "storage" carbon?

Reply:
It's a generic store, without a home/organ. And in principle includes sugars and starch. I guess extending to 'n' storage pools is a plausible use-case of this code...